config/cc/gcc.in

```diff
@@ -54,6 +54,15 @@ config CC_GCC_ENABLE_CXX_FLAGS
       Note: just pass in the option _value_, that is only the part that goes
       after the '=' sign.
 
+config CC_GCC_EXTRA_LIBSTDCXX
+    bool "Re-enable libstdcxx"
+    default n
+    depends on LIBC_AVR_LIBC
+    select CC_CORE_PASS_2_NEEDED
+    help
+      libstdcxx is normaly disabled on avr systems, so yout can define your on datatypes
+      like <type-traits>. This re-enable this.
```

Comment:
Couple of bit of a typos here "normaly" and "yout". I think perhaps we can write this as

```
      libstdcxx is normally disabled on avr systems due to size limitations. Set this option to re-enable
      libstdcxx support.
```